minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-resources/src/main/resources/conf/bootstrap.conf

```diff
@@ -28,6 +28,16 @@ conf.dir=./conf
 # How long to wait after telling MiNiFi to shutdown before explicitly killing the Process
 graceful.shutdown.seconds=20
 
+# The location for the configuration file
+nifi.minifi.config=./conf/config.yml
+
+# Notifiers to use for the associated agent, comma separated list of class names
+#nifi.minifi.notifier.components=org.apache.nifi.minifi.bootstrap.configuration.FileChangeNotifier
+
+# File change notifier configuration for path of the file to monitor and the associated polling period
+#nifi.minifi.notifier.file.config.path=
```

Comment:
Yep, we can adjust.  It's just a matter of how we convey to the listener that there is not an input stream/file such that the listener shouldn't save.  Let me refactor a bit and push another commit out.